sched/scheduler/jobRunner_test.go

```diff
@@ -40,8 +36,7 @@ func Test_RunTask_PanicWhenWritingStartTaskReturnsFatalError(t *testing.T) {
 	mockCtrl := gomock.NewController(t)
 	defer mockCtrl.Finish()
 
-	// ensures node does not execute task when a fatal error occurs logging starttask
-	nodeMock := cm.NewMockNode(mockCtrl)
+	worker := worker.NewNoopWorker()
```

Comment:
Hmm, thinking about this more: why? Why is it important that the task did not get sent to the worker?

I agree that testing is super important, but also that testing the wrong things leads us down a path that makes it harder to do all the testing we could.